### ISSUE TITLE
evalution on another deepSense model?

### ISSUE THREAD

**Issue by kaixinbuyu** [yscacaca/DeepSense#3]
#create train model
logits = deepSense(batch_feature, True, name='deepSense')

#create eval model
logits_eval = deepSense(batch_eval_feature, False, reuse=True, name='deepSense')

the eval loss is always at 0.2.
why evalution on another deepSense model?so the eval loss does not increase.

**Comment by yscacaca** [comment]
Yes, these are two models with the same parameter, because we set "resue=True" in the eval model.  Dropout operations are different for training and evaluation.

**Comment by kaixinbuyu** [comment]
think you! I found other people use it in the same way:

"I am setting up two graphs for training and evaluation using `tf.variable_scope('my_model', reuse=True)` to make it share the weights of the training model."
From: https://groups.google.com/a/tensorflow.org/forum/#!topic/discuss/4LVyal5kspg
